vendor/bat-native-ledger/src/bat/ledger/internal/ledger_impl.cc

```diff
@@ -106,11 +111,32 @@ void LedgerImpl::OnWalletInitializedInternal(
 void LedgerImpl::Initialize(ledger::InitializeCallback callback) {
   DCHECK(!initializing_);
   initializing_ = true;
+
+  initialize_callback_ = callback;
+
+  confirmations::_environment = ledger::_environment;
+  confirmations::_is_debug = ledger::is_debug;
+
+  bat_confirmations_.reset(
+      confirmations::Confirmations::CreateInstance(ledger_client_));
+
+  auto initialize_step_2_callback =
+      std::bind(&LedgerImpl::InitializeStep2, this, _1);
```

Comment:
let's add callback to bind so that we don't need `initialize_callback_` and it's just passed to `InitializeStep2`

Reply:
Fixed